File: core/src/main/java/net/yan100/compose/core/models/ReFlushTokenModel.java
package net.yan100.compose.core.models;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.time.LocalDateTime;

/**
 * 过期时间令牌
 *
 * @author TrueNine
 * @since 2022-12-20
 */
@Getter
@Setter
@ToString
@EqualsAndHashCode
public class ReFlushTokenModel {
  private String userId;
  private String deviceId;
  private String loginIpAddr;
  private LocalDateTime issueAt;
  private LocalDateTime expireTime;
}

File: core/src/main/kotlin/net/yan100/compose/core/models/ReFlushTokenModel.kt
package net.yan100.compose.core.models

import lombok.EqualsAndHashCode
import lombok.Getter
import lombok.Setter
import lombok.ToString
import java.time.LocalDateTime

/**
 * 过期时间令牌
 *
 * @author TrueNine
 * @since 2022-12-20
 */
class ReFlushTokenModel {
  var userId: String? = null
  var deviceId: String? = null
  var loginIpAddr: String? = null
  var issueAt: LocalDateTime? = null
  var expireTime: LocalDateTime? = null
}
